atlp-rwanda/ecommerce-app-champions-fe
# #73Feature: password change prompt

### what does this PR do?

- user is prompted to change password x amount of time

### description

- implement change password prompt
- how should it be manually tested
- clone the repo using git clone repo's URL
- change branch git checkout ft-password-change-prompt
- create .env file and paste variables from .env.example
- run npm run dev
- make sure that you are registered
- navigate to login and paste in your credentials and if your password has expired there will pop up an alert
- click on that alert and navigate to your email
- check for the password expiry email
- click on it to update your password

<img width="1440" alt="Screenshot 2023-05-03 at 14 43 25" src="https://user-images.githubusercontent.com/87751814/235920169-9c2d1a85-75da-49a7-9e1f-4d1f58724e3d.png">
